package.json

```diff
@@ -25,12 +26,22 @@
   ],
   "packageManager": "yarn@4.2.1",
   "devDependencies": {
-    "@commitlint/cli": "^18.6.1",
-    "@commitlint/config-conventional": "^18.6.2",
+    "@commitlint/cli": "^19.3.0",
+    "@commitlint/config-conventional": "^19.2.2",
+    "@types/chai": "^4",
+    "@types/mocha": "^10",
     "changelogithub": "patch:changelogithub@npm%3A0.13.3#~/.yarn/patches/changelogithub-npm-0.13.3-1783949906.patch",
     "czg": "^1.9.1",
     "dprint": "^0.46.3",
-    "husky": "^9.0.11",
-    "lint-staged": "^15.2.2"
-  }
+    "husky": "4",
```

Comment:
@YashBit why did you downgrade `husky`?
I realised this is what broke the CI
https://github.com/privacy-scaling-explorations/zk-kit.noir/actions/runs/10564012800/job/29265649521?pr=8#step:4:36